mitiq/executor/executor.py

```diff
@@ -346,7 +346,7 @@ def _call_executor(
 
         if self.can_batch:
             self._quantum_results.extend(result)
-            self._executed_circuits.extend(to_run)
+            self._executed_circuits.extend(to_run)  # type: ignore
```

Comment:
What's with all the type ignore comments? Why are they needed the the new QPROGRAM addition?